web/styles/_sidebar.scss

```diff
@@ -103,77 +142,32 @@ button {
     z-index: 2000;
     top: 0;
     width: 280px; /* works all the way down to an iphone 4 */
+    // Set a max width for narrow layouts that accounts for margin.
+    max-width: calc(100% - 20px);
```

Comment:
This is the only functional change as during my testing very narrow layouts resulted in the sidenav covering content when not open.